integration/sql/iface-java/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "openlineage_sql_java"
+description = "Java interface for the Rust OpenLineage lineage extraction library"
+version = "0.15.0"
+edition = "2021"
```

Comment:
Is this an edition of a package?

Reply:
Rust language edition. Basically equivalent to Java versions, like 8 or 11: https://doc.rust-lang.org/edition-guide/editions/index.html